dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/event/ChangeLogRequestParams.java

```diff
@@ -27,26 +27,28 @@
  */
 package org.hisp.dhis.webapi.controller.tracker.export.event;
 
+import java.util.ArrayList;
 import java.util.List;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 import org.hisp.dhis.common.OpenApi;
 import org.hisp.dhis.fieldfiltering.FieldFilterParser;
 import org.hisp.dhis.fieldfiltering.FieldPath;
+import org.hisp.dhis.webapi.controller.event.webrequest.OrderCriteria;
 
 @OpenApi.Shared(name = "ChangeLogRequestParams")
 @OpenApi.Property
 @Data
 @NoArgsConstructor
 public class ChangeLogRequestParams {
 
-  static final String DEFAULT_FIELDS_PARAM = "createdBy, change, createdAt";
+  static final String DEFAULT_FIELDS_PARAM = "change,createdAt,createdBy";
 
-  @OpenApi.Property(defaultValue = "1")
-  private Integer page;
+  private int page = 1;
 
-  @OpenApi.Property(defaultValue = "50")
-  private Integer pageSize;
+  private int pageSize = 50;
 
   private List<FieldPath> fields = FieldFilterParser.parse(DEFAULT_FIELDS_PARAM);
+
+  private List<OrderCriteria> order = new ArrayList<>();
```

Comment:
why are you adding order? Let's get pagination done first without order if possible